### ISSUE TITLE
Error with mini.nvim -> `statusline.lua` on fresh install

### ISSUE THREAD

**Issue by Maruncho** [nvim-lua/kickstart.nvim#1126]
## Describe the bug
First time neovim user, excuse the lack of knowledge.
I followed the instructions on the README.md, ran the git clone, ran nvim, everything went smoothly, but this error appeared on the bottom.

E5108: Error executing lua ....local/share/nvim/lazy/mini.nvim/lua/mini/statusline.lua:656: attempt to call field 'is_e
nabled' (a nil value)
stack traceback:
        ....local/share/nvim/lazy/mini.nvim/lua/mini/statusline.lua:656: in function 'diagnostic_is_disabled'
        ....local/share/nvim/lazy/mini.nvim/lua/mini/statusline.lua:327: in function 'section_diagnostics'
        ....local/share/nvim/lazy/mini.nvim/lua/mini/statusline.lua:611: in function <....local/share/nvim/lazy/mini.nv
im/lua/mini/statusline.lua:606>

It appears that `vim.diagnostics.is_enabled` does not exist. The error comes from this if block, precisely the 1st branch (2nd line):

      if vim.fn.has('nvim-0.10') == 1 then
        H.diagnostic_is_disabled = function(_) return not vim.diagnostic.is_enabled({ bufnr = 0 }) end
      elseif vim.fn.has('nvim-0.9') == 1 then
        H.diagnostic_is_disabled = function(_) return vim.diagnostic.is_disabled(0) end
      else
        H.diagnostic_is_disabled = function(_) return false end
      end

I **think** I fixed the bug by replacing it with `is_disabled` and removing the `not` on the front. After this fix, the message was gone.
I checked the docs and contrary to what the error shows , `is_enabled` is present while `is_disabled` was not mentioned anywhere, so it's probably something to do with neovim itself and not kickstart.nvim.

## To Reproduce

1. Fresh install neovim, install recipes for WSL, install kickstart. Run nvim, finish installing stuff, see error message.

## Desktop

- OS: WSL on Windows 10
- Terminal: bash

## Neovim Version

```
NVIM v0.10.0-dev
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
```

**Comment by abeldekat** [comment]
This *might* be caused by using an older development version of Neovim, as was the case in  #1085 

See [this](https://github.com/nvim-lua/kickstart.nvim/issues/1085#issuecomment-2288349945) comment

The suggested solution:

> ... Updating to released 0.10 (or relatively late development build) will fix this.

**Comment by Maruncho** [comment]
I installed nvim with the tarbal, no errors are shown. It seems the neovim-ppa is outdated (the Ubuntu install instructions).
